deploy/single_filter.yaml

```diff
@@ -0,0 +1,23 @@
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
```

Comment:
These are more like test fixtures. We should not keep them in `deploy` but rather a `test` or `test/fixtures` directory.

Reply:
That's true, I will move these